src/RuleList.js

```diff
@@ -136,13 +129,14 @@ export default class RuleList {
    */
   update(name?: string | Object, data?: Object): void {
```

Comment:
It is optional here, because we have arguments an optional first argument, which results in potentially optional second argument (now I think it was a bad idea to make optional first argument, given the flow typing)

Reply:
Is there a way to express this:

if name was passed, data is not optional, if only name is passed and its an object - it is the data object.

Reply:
It will look really unmaintainable. Can't we just use named arguments?